architecture/core_concepts/routes.adoc

```diff
@@ -196,17 +196,17 @@ variables on the *deployment config* for the router to alter its configuration.
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named tls.crt. If tls.crt is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if DEFAULT_CERTIFICATE or DEFAULT_CERTIFICATE_PATH are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if DEFAULT_CERTIFICATE is not specified.
-|`*EXTENDED_VALIDATION*` | true | Whether to perform an additional extended validation step on all routes admitted by this router.
+|`*EXTENDED_VALIDATION*` | true | If __"true"__,perform an additional extended validation step on all routes admitted by this router.
```

Comment:
@gaurav-nelson The markup might not be best here. As per the guidelines ( https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference ) a backtick for the value would be better.

This applies to a few other parts of this section as well.